### PR TITLE
Improve Blockchain Roadmap

### PR DESCRIPTION
**Pull request by offensive-vk**
> [!TIP]
> Blockchain Roadmap - https://roadmap.sh/blockchain

- Fix Topic Link Anchors
- Fix Topic Link Types
- Fix Topic Link Text
- Format Every Content
- Add More Links
- Add Topic Content
- Fix Deprecated Content
- Remove Deprecated Topic Links

This PR is intended to improve the Blockchain Roadmap as it lacks fresh or *refreshing* content and few more topic links.

Status: *Finished* ⚒️

### CODE REVIEW

**Comment by dansholds** [comment]
@offensive-vk theres no need to open 3 draft PRs at once

**Comment by dansholds** [comment]
> I like draft PRs. And I am helping you right, (I'm gonna open more real soon)

You can make the changes without opening Draft PRs, just create the PR when it's complete.

File: src/data/roadmaps/blockchain/content/ethersjs@i-ltfXPTCu3WaBo-xaN05.md [hunk 1]
```diff
@@ -4,5 +4,4 @@ The ethers.js library aims to be a complete and compact library for interacting
 
 Visit the following resources to learn more:
 
-- [@article@Ethers.js Documentation](https://docs.ethers.io/)
-- [@feed@Explore top posts about JavaScript](https://app.daily.dev/tags/javascript?ref=roadmapsh)
```
Comment (dansholds) [inline src/data/roadmaps/blockchain/content/ethersjs@i-ltfXPTCu3WaBo-xaN05.md:8]: Don't removed partnered content

File: src/data/roadmaps/blockchain/content/general-blockchain-knowledge@bA4V_9AbV3uQi3qrtLWk0.md [hunk 2]
```diff
@@ -1,7 +1,10 @@
-# Blockchain general knowledge
+# Blockchain
+
+A blockchain is a decentralized, distributed ledger technology that records transactions across many computers in such a way that the registered transactions cannot be altered retroactively. This technology is the backbone of cryptocurrencies like Bitcoin and Ethereum, but its applications extend far beyond digital currencies.
 
 Visit the following resources to learn more:
 
+- [@article@What is Blockchain?](https://www.wired.com/story/guide-blockchain/)
```
Comment (dansholds) [inline src/data/roadmaps/blockchain/content/general-blockchain-knowledge@bA4V_9AbV3uQi3qrtLWk0.md:7]: Change to either "What is the Blockchain" or "What is a Blockchain"

**Review by dansholds — CHANGES_REQUESTED** [review]

**Comment by offensive-vk** [comment]
Sure thing, dan
I'll review at night.

**Comment by offensive-vk** [comment]
Hey, @dansholds, what's this hacktoberfest thing and what about its label in here?

File: src/data/roadmaps/blockchain/content/applications-and-uses@WD2JH4X4tEE4J0W0XFQ_4.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
-# Application and uses of Blockchain technology
+# Application and Uses of Blockchain technology
```
[the remark this answers is not in the document]
Comment (offensive-vk) [reply src/data/roadmaps/blockchain/content/applications-and-uses@WD2JH4X4tEE4J0W0XFQ_4.md:1]: Sure.

**Comment by dansholds** [comment]
> Hey, @dansholds, what's this hacktoberfest thing and what about its label in here?

https://hacktoberfest.com/

**Review by dansholds — APPROVED** [review]

**Comment by offensive-vk** [comment]
bro 😭😭

**Pull request merged**